### ISSUE TITLE
Subheading block: text lacks sufficient contrast

### ISSUE THREAD

**Issue by sarahmonster** [WordPress/gutenberg#8716]
The subheading block uses a light grey for the text, which lacks sufficient contrast against the white background:

<img width="685" alt="screenshot 2018-08-08 15 05 29" src="https://user-images.githubusercontent.com/376315/43842107-8274aa12-9b1c-11e8-9449-6a1712a20a30.png">

> The color combination #9199a1/#ffffff has a contrast ratio of 2.89, which is not sufficient. At this size, you will need a ratio of at least 4.5. 

For better legibility, this should probably be increased.

**Comment by abrightclearweb** [comment]
Agreed on this. Does Subheading have a semantic meaning to assistive technologies? 

It is marked up as `<p class="wp-block-subhead">Subheading</p>` so I'm guessing not.

**Comment by youknowriad** [comment]
Subheading block is deprecated.
